### ISSUE TITLE
Angular 12

### ISSUE THREAD

**Issue by Novinyo** [grzegorz103/ngx-chess-board#192]
This does not work with Angular version 12. I tried all your versions none is working with it

**Comment by grzegorz103** [comment]
Hi. Is there any error when you start the app?

**Comment by yousefalaqra** [comment]
I'm using Angular 12 (i have no choice, I had to downgrade from13 it) anyways, I might be facing the same issue. Which is:

```
Error: node_modules/ngx-chess-board/lib/engine/drag/end/drag-end-processor.d.ts:1:28 - error TS2307: Cannot find module '@angular/cdk/drag-drop' or its corresponding type declarations.

1 import { CdkDragEnd } from '@angular/cdk/drag-drop';
                             ~~~~~~~~~~~~~~~~~~~~~~~~
````

**Comment by grzegorz103** [comment]
@yousefalaqra try please install angular cdk ```npm i @angular/cdk```
SInce some Angular version it got deleted as i remember
Let me know if this helped

**Comment by yousefalaqra** [comment]
@grzegorz103 Yes, it works right now! thank you! I would be happy to contribute to your source code, let me know what do you need and I would be love help.

Anyways, I've checked the source and I see that the angular CDK is already installed, why do I need to install it again in my project?

**Comment by grzegorz103** [comment]
Thanks. This lib has already more things implemented than i ever planned so feel free to push anything comes to your mind :)
Angular cdk is added as peer dependency to the lib, so it isn't installed automatically with the lib. It could be added as normal dependency instead of peer, but im not sure if it is good practice to manage dependencies that way in angular

**Comment by yousefalaqra** [comment]
I think there's a performance issue that I'm considering working on it. I'll investigate it, report it, and fix it
